### DISCUSSION TITLE
Create file selection when creating a new file.

### DISCUSSION THREAD

**Discussion by 0xWDG** [AuroraEditor/AuroraEditor#121] [Ideas]
Create file selection when creating a new file.

See: #27

**Comment by KaiTheRedNinja** [comment]
File extension inference is our current solution to this problem. We may chose to go the xcode route and offer code templates as well in a pop-up, but thats for the future.

**Comment by nanashili** [comment]
We might want to make the textfield a first responder so we can allow the field to edited. I added a reference below for some help.

https://stackoverflow.com/a/49336194/17483038

**Comment by KaiTheRedNinja** [comment]
https://github.com/AuroraEditor/AuroraEditor/blob/473dbeddb6ea33a7f3b325368d642d3feaa4ce5c/AuroraEditor/Base/NavigatorSidebar/Model/ProjectNavigator/OutlineTableViewCell.swift#L185

This is dummy code where in the future the feature could be implemented

**Comment by nanashili** [comment]
The following PR handles this idea: https://github.com/AuroraEditor/AuroraEditor/pull/149
